src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Tools/project.json

```diff
@@ -1,6 +1,6 @@
 {
-  "version": "1.0.0-*",
-  "description": "Razor precompilation",
+  "version": "1.1.0-*",
+  "description": "Razor view compilation",
```

Comment:
As far as the difference between .Design and .Tools, we should probably say *something* so people know what to reference from where. Do we ever do anything like that?